pandora_console/godmode/agentes/planned_downtime.editor.php

```diff
@@ -345,9 +352,11 @@
 					__('From:') .
 					"</td>
 				<td>".
-				html_print_input_text ('once_date_from', $once_date_from, '', 10, 10, true) .
+				html_print_input_text_extended ('once_date_from', $once_date_from, '', 10, 10, true,
+					$disabled_in_execution, '', $disabled_in_execution ? "disabled=disabled" : '') .
 				ui_print_help_tip(__('Date format in Pandora is year/month/day'), true) .
-				html_print_input_text ('once_time_from', $once_time_from, '', 9, 9, true) .
+				html_print_input_text_extended ('once_time_from', $once_time_from, '', 9, 9, true, 
```

Comment:
function html_print_input_text with disabled arg generates input element with readonly attribute.
However jquery datepicker ignore readonly attribute because readonly attribute has a focus event, in contrast with disable attribute remove focus events.
This datepicker behavior is base on [by design](http://bugs.jqueryui.com/ticket/8001), so won't fixed.
I tried workarounds for datepicker, one is [this answer on stackoverflow](http://stackoverflow.com/questions/7812063/jquery-datepicker-readonly#answer-8315058). However these has no effects on timepicker plugin.